src/components/Footer/Footer.tsx

```diff
@@ -1,13 +1,15 @@
+/* eslint-disable react/prop-types */
```

Comment:
We can probably disable prop-types completely, since TypeScript makes it generally unnecessary?

Reply:
They're still useful during the gradual migration imo